src/api/chats/post.ts

```diff
@@ -0,0 +1,27 @@
+import {Response} from 'express';
+import {HttpError} from '../../common/errors';
+import {wrapper} from '../../tools/wrapper.helpers';
+import {IEntityRequest} from '../../types';
+import {MessageEntity} from '../../db/entities/message.entity';
+import {ChatEntity} from "../../db/entities/chat.entity";
+
+export const createMessage = wrapper(async (req: IEntityRequest<ChatEntity>, res: Response) => {
+    const {entity: chat, user} = req;
+    const chatMembers = await chat.members;
+    const checkChatMember = chatMembers.some(chatMember => chatMember.userId === user.id)
+
```

Comment:
U have identical lines of code at method `getChatMessages`.Remove duplicate code,please, and don't forget about DRY principle.